Java/JDI/jdi-cucumber-test-generator/src/main/webapp/WEB-INF/static/js/suit/suitMain.js

```diff
@@ -123,8 +165,8 @@ function getSuitInfoWithOutCleanCases(suitId){
                                             .text(response.cases[i].description)
                                         ).append($('<td>')
                                             .text(response.cases[i].priority)
-                                        ).append($('<td>')
-                                            .text(response.cases[i].tags)
+                                        ).append($('<td >').attr("id", "i-have-a-tooltip").attr("data-description", tagsToString)
```

Comment:
Ask Kostya about "i-have-a-tooltip" class, because yestraday he said that he won't use it, so maybe you need to delete.